src/components/CardTable/CardTable.jsx

```diff
@@ -109,14 +109,16 @@ export default class CardTable extends Component {
   }
 
   get largestTableWidth () {
-    const tableNodes = Object.values(this.tableNodes)
+    const tableNodes = Object.values(this.tableNodes || {})
     const tableWidths = tableNodes.map(tableNode => tableNode.clientWidth)
 
     return Math.max(...tableWidths)
   }
 
   get parentWidth () {
-    return getElementContentWidth(this.containerNode.parentNode)
+    const ZERO = 0
+
+    return this.containerNode ? getElementContentWidth(this.containerNode.parentNode) : ZERO
```

Comment:
Yeh should be ok but it would mean we need to handle it in the `getElementContentWidth` with something like
```javascript
getElementContentWidth (element) {
  if (!(element instanceof Element)) {
    const ZERO = 0

    return ZERO
  }
.....
```
As I said I'm unable to get this error again, so it might not be necessary. But let me know what you think of this solution.